specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2023-08-01/service.json

```diff
@@ -911,6 +911,60 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/removeInfra": {
+      "post": {
+        "tags": [
+          "ReplicationInfrastructure"
+        ],
+        "summary": "Removes the appliance's infrastructure under the fabric.",
+        "operationId": "ReplicationInfrastructure_Delete",
```

Comment:
Can we rename this to RemoveInfra

Reply:
The up to date operationId is "ReplicationFabrics_RemoveInfra". Here, the parent entity is ReplicationFabrics and the operation is RemoveInfra following other API conventions.